### PR TITLE
Specific version of PS where it is available

### PR DESCRIPTION
**Pull request by msaustral**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.x / 8.x
| Description?  | Please be specific when describing the PR. What did you add, why did you submit it?
| Fixed ticket? | Fixes #{issue number here} if there is a related issue

### CODE REVIEW

**Review by tleon — APPROVED** [review]

File: development/components/form/types-reference/number-min-max-filter.md [hunk 1]
```diff
@@ -4,6 +4,7 @@ title: NumberMinMaxFilterType
 
 # NumberMinMaxFilterType
 
+<h1>Available from PS 1.7.7.x and UP</h1>
```
Comment (thomasnares) [inline development/components/form/types-reference/number-min-max-filter.md]:
```suggestion
```

File: development/components/form/types-reference/number-min-max-filter.md [hunk 2]
```diff
@@ -4,6 +4,7 @@ title: NumberMinMaxFilterType
 
 # NumberMinMaxFilterType
```
Comment (thomasnares) [inline development/components/form/types-reference/number-min-max-filter.md]:
```suggestion
# NumberMinMaxFilterType {{< minver v="1.7.7.x" title="true" >}}
```

**Review by thomasnares — COMMENTED** [review]
Thanks @msaustral, it should be like this (my suggestion)

**Review by kpodemski — CHANGES_REQUESTED** [review]
Hello @msaustral 

Please follow @thomasnares suggestions.

**Review by thomasnares — APPROVED** [review]
thanks @msaustral 👍

**Review by kpodemski — APPROVED** [review]

**Pull request merged**